src/openrct2/world/Sprite.cpp

```diff
@@ -847,7 +880,7 @@ void litter_create(const CoordsXYZD& litterPos, int32_t type)
     litter->sprite_identifier = SpriteIdentifier::Litter;
     litter->type = type;
     litter->MoveTo(offsetLitterPos);
```

Comment:
Are you going to go through every MoveTo and remove the invalidates?

Reply:
Yeah I missed that one.